src/components/SurveyAnswer/ChoiceAnswer.tsx

```diff
@@ -0,0 +1,43 @@
+import SimpleBar from 'simplebar-react';
+
+import { SurveyAnswerProps } from '.';
+
+import 'simplebar-react/dist/simplebar.min.css';
+
+const ChoiceAnswer = ({
+  question,
+  currentResponse,
+  setResponse,
+}: SurveyAnswerProps) => {
+  const onAnswerSelect = (answerId: string) => {
+    setResponse({ questionId: question.id, answers: [{ id: answerId }] });
+  };
+
+  const answerStateClass = (answerId: string) => {
+    const isActiveAnswer = currentResponse?.answers[0].id === answerId;
+
+    if (isActiveAnswer) {
+      return 'border-y border-gray-300 text-white font-bold';
+    }
+
+    return 'text-gray-400';
+  };
```

Comment:
Should we add some tests to cover the answerStateClass as well as the onAnswerSelect logic? 🤔

Reply:
The onAnswerSelect logic is already covered in `click on answer` test case. So I added the test to cover the active and inactive states here [6f9cb5f](https://github.com/hanam1ni/react-surveyor/pull/82/commits/6f9cb5f85fb04d9561ba950636c72bc30ad34d82)

Reply:
Oh yes, true! Thanks!